### PR TITLE
Adds Styling to Lecture List

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr adds bootstrap to the lecture list page.

Screenshots:
https://screenshot.googleplex.com/ktrmGDCmWXJ.png
https://screenshot.googleplex.com/6vuUMdbWNVP

### CODE REVIEW

File: src/main/webapp/lecture-list.html [hunk 1]
```diff
@@ -1,18 +1,85 @@
...
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addLectureModal">
+      Add a Lecture
+    </button>
+  </div>
+</body>
+
+<footer id="footer">
+  <div class="container">
+    <p class="text-center">
+      <a href="#" class="link">About</a> | <a href="#" class="link">Feedback</a> |
```
Comment (n-wach) [inline src/main/webapp/lecture-list.html:41]: Can you make About redirect to "/"?  And Feedback redirect to our github issues page?

Comment (jaysaleh) [reply src/main/webapp/lecture-list.html]: added

File: src/main/webapp/list.html [hunk 2]
```diff
@@ -1,18 +1,86 @@
...
+
+<body>
+  <nav class="navbar navbar-light">
+    <h2 id="header-text">ZoomTube</h2>
+    <button type="button" class="btn btn-outline-primary">Logout</button>
+  </nav>
+
+  <div id="list-cont">
+    <div id="text-cont">
+      <h2 class="text-center">Active Lectures</h2>
```
Comment (billie-cat) [inline src/main/webapp/list.html:23]: I find the word "Active" a bit weird. Would "Available Lectures" make more sense? I do not feel strongly against this. If the others like it you can keep it as is.

Comment (jaysaleh) [reply src/main/webapp/list.html]: I agree, made the fix

**Pull request merged**